tests/QAN/timerange_test.js

```diff
@@ -82,3 +84,116 @@ Scenario(
+    qanOverview.selectRow(2);
+    I.click(qanOverview.buttons.copyButton);
+
+    const url = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+    const toTimeFromUrl = qanOverview.getToTimeFromUrl(url);
+
+    assert.ok(Math.abs(dateTime - toTimeFromUrl) < 30000, 'Difference between moment time and first copied time must be less then half of minute');
+
+    I.wait(30);
+    I.refreshPage();
+    qanOverview.waitForOverviewLoaded();
+    I.click(qanOverview.buttons.copyButton);
+
+    const url2 = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+    const toTimeFromUrl2 = qanOverview.getToTimeFromUrl(url2);
+
+    assert.ok(Math.abs(toTimeFromUrl - toTimeFromUrl2) < 60000, 'Difference between moment time and second copied time must be less then one minute');
+
+    I.openNewTab();
+    I.amOnPage(url);
+    qanOverview.waitForOverviewLoaded();
+    I.waitForVisible(qanOverview.getSelectedRowLocator(2));
+  },
+);
+
+Scenario(
+  'PMM-T1140 - Verify relative time range copy URL from browser @qan',
+  async ({ I, qanOverview }) => {
+    const url = await I.grabCurrentUrl();
+    const fromToString = qanOverview.getFromToTimeFromUrl(url);
+
+    I.wait(60);
+    I.openNewTab();
+    I.amOnPage(url);
+    qanOverview.waitForOverviewLoaded();
+
+    const url2 = await I.grabCurrentUrl();
+    const fromToString2 = qanOverview.getFromToTimeFromUrl(url2);
+
+    assert.equal(fromToString, fromToString2, `The time range ${fromToString} is NOT the same you were seeing in previously tab ${fromToString2}`);
+  },
+);
+
+Scenario(
+  'PMM-T1141 - Verify specific time range by new button to copy QAN URL @qan',
+  async ({ I, adminPage, qanOverview }) => {
+    const dateTime = moment();
+    const to = dateTime.format('YYYY-MM-DD HH:mm:ss');
+    const from = moment(dateTime).subtract(1, 'hours').format('YYYY-MM-DD HH:mm:ss');
+    const fromToString = `&from=${moment(from).format('x')}&to=${moment(to).format('x')}`;
+
+    adminPage.setAbsoluteTimeRange(from, to);
+    qanOverview.waitForOverviewLoaded();
+    I.seeInCurrentUrl(fromToString);
+    I.click(qanOverview.buttons.copyButton);
+
+    const url = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+
+    I.openNewTab();
+    I.amOnPage(url);
+    I.seeInCurrentUrl(fromToString);
+  },
+);
+
+Scenario(
+  'PMM-T1142 - Verify that the table page and selected query are still the same when we go on copied link by new QAN CopyButton @qan',
+  async ({ I, qanPagination, qanOverview }) => {
+    I.click(qanPagination.buttons.nextPage);
+    qanOverview.selectRow(2);
+    I.click(qanOverview.buttons.copyButton);
+
+    const url = await I.grabTextFrom(qanOverview.elements.clipboardLink);
+
+    I.openNewTab();
+    I.amOnPage(url);
+    qanOverview.waitForOverviewLoaded();
+    // this check will need to be uncommented after tasks 9480 and 9481 are ready
+    // qanPagination.verifyActivePage(2);
+    I.waitForVisible(qanOverview.getSelectedRowLocator(2), 20);
```

Comment:
@ElenaFedichkina  could you also make sure the Detail section is also opened automatically when opened in new tab

Reply:
ok